### PR TITLE
[BESU-100] Made client library content easier to find.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## PR description
Make it easier to find client library content.

## Fixed Issue(s)
BESU-100

### CODE REVIEW

File: docs/Concepts/Client-Libraries.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+description: Hyperledger Besu client libraries 
+<!--- END of page meta data -->
+
+# Client Libraries
+
+Client libraries such as web3.js, web3j, or ethereumj are used by Dapps to forward JSON-RPC requests to Hyperledger Besu. Any
```
Comment (MadelineMurray) [inline docs/Concepts/Client-Libraries.md]: Suggest adding links on the client libraries to their doc or github repos.

File: docs/Concepts/Client-Libraries.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+client library that implements core Ethereum RPC methods can be used with Besu.
+
+The web3.js-eea library can be used with Besu for [privacy features](../Concepts/Privacy/Privacy-Overview.md).
```
Comment (MadelineMurray) [inline docs/Concepts/Client-Libraries.md]: The web3.js-eea library can be used -> Use the web3.js-eea library with Besu .... 
Edit to make active. 

Add a link on web3.js-eea library to /HowTo/Interact/Client-Libraries/eeajs/

File: docs/Concepts/Client-Libraries.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+![Client Libraries](../images/Hyperledger-Besu-Client-Libraries.png)
+
+Client libraries can also be used to [create signed transactions](../HowTo/Develop-Dapps/Use-web3js.md). Use the web3.js-eea library to [create and send private transactions](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md).
```
Comment (MadelineMurray) [inline docs/Concepts/Client-Libraries.md]: Would this be better as a bulleted list? eg, 

Use client libraries to: 
* Create signed transactions 
* Create and send private transactions

File: docs/HowTo/Develop-Dapps/Use-web3js.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+```bash tab="Command"
+node create_value_raw_transaction.js <YOUR JSON-RPC HTTP ENDPOINT>
+```
+
+```bash tab="Example"
+node create_value_raw_transaction.js http://localhost:32770/jsonrpc
+```
+
+!!! tip
+    The default JSON-RPC HTTP endpoint for `create_value_raw_transaction.js` is `http://localhost:8545`.
```
Comment (lucassaldanha) [inline docs/HowTo/Develop-Dapps/Use-web3js.md]: It took me some time to understand this tip. Maybe rephrase it like this:
```
The default JSON-RPC HTTP endpoint for `create_value_raw_transaction.js` is `http://localhost:8545`.
If Besu is listening on `http://localhost:8545`, you can run the command without specifying the endpoint:
`node create_value_raw_transaction.js`
```

File: docs/Concepts/Client-Libraries.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Concepts/Client-Libraries.md]: Done.

File: docs/Concepts/Client-Libraries.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Concepts/Client-Libraries.md]: Done.

File: docs/Concepts/Client-Libraries.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Concepts/Client-Libraries.md]: I was two-minded about this. I'll make the change since you also recommending this.

File: docs/HowTo/Develop-Dapps/Use-web3js.md (hunk 4 above)
Comment (bgravenorst) [reply docs/HowTo/Develop-Dapps/Use-web3js.md]: Done.
